pixel/style.css

```diff
@@ -10,15 +10,31 @@ Requires PHP: 5.7
 Version: 0.0.1
 License: GNU General Public License v2 or later
 License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
-Template: 
+Template:
 Text Domain: pixel
 Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
 */
 
-/* 
+/*
  * Custom class to add shadow to blocks
- * Need this until box shadow support added to Gutenberg here: https://github.com/WordPress/gutenberg/pull/43184 
+ * Need this until box shadow support added to Gutenberg here: https://github.com/WordPress/gutenberg/pull/43184
  */
 .pixel-shadow {
-    box-shadow: 5px 5px 0px -2px var(--wp--preset--color--background), 5px 5px var(--wp--preset--color--foreground);
+	box-shadow: 5px 5px 0 -2px var(--wp--preset--color--background), 5px 5px var(--wp--preset--color--foreground);
+}
+
+.wp-block-post-title a {
+	text-underline-offset: 8px;
+	text-decoration-thickness: 2px;
+}
+
+.wp-block-post-title a:hover,
```

Comment:
Just confirming that this cannot be set via theme.json pseudo states yet, right?